setup.py

```diff
@@ -1124,11 +1117,58 @@ def print_box(msg):
 
 
 def main():
-    if BUILD_LIBTORCH_WHL and BUILD_PYTHON_ONLY:
+    global BUILD_LIBTORCH_WHL
+    global BUILD_PYTHON_ONLY
+    global PACKAGE_NAME
+
+    BUILD_LIBTORCH_WHL = os.getenv("BUILD_LIBTORCH_WHL", "0") == "1"
```

Comment:
Nit: Is there a reason to bring the initialization `BUILD_LIBTORCH_WHL = os.getenv("BUILD_LIBTORCH_WHL", "0") == "1"` and `BUILD_PYTHON_ONLY = os.getenv("BUILD_PYTHON_ONLY", "0") == "1"` here instead of where they are defined above?  It's better to defined and init a variable in the same place if possible IMO